src/Microsoft.ML.Data/Transforms/Hashing.cs

```diff
@@ -124,17 +130,18 @@ private static VersionInfo GetVersionInfo()
                 modelSignature: "HASHTRNS",
                 // verWrittenCur: 0x00010001, // Initial
                 // verWrittenCur: 0x00010002, // Invert hash key values, hash fix
-                verWrittenCur: 0x00010003, // Uses MurmurHash3_x86_32
-                verReadableCur: 0x00010003,
+                //verWrittenCur: 0x00010003, // Uses MurmurHash3_x86_32
+                verWrittenCur: 0x00010004, // Add Combine option
+                verReadableCur: 0x00010004,
```

Comment:
We still haven't released a version with MurmurHash3. Do we need to increment this if we make sure to include it in the v1.5 release?

Reply:
Technically users may create and save models containing this transformer once the PR that changed the version to 0x00010003 was merged. This most likely didn't happen, but to be on the safe side I think it's better to increment the version again. 

---
In reply to: [424565803](https://github.com/dotnet/machinelearning/pull/4828#discussion_r424565803) [](ancestors = 424565803)